tensorflow/contrib/rnn/python/ops/rnn_cell.py

```diff
@@ -661,6 +663,109 @@ def _make_tf_features(self, input_feat):
       freq_inputs.append(cur_input)
     return freq_inputs
 
+class ESNCell(rnn_cell.RNNCell):
+  """Echo State Network Cell.
+
+    Based on http://www.faculty.jacobs-university.de/hjaeger/pubs/EchoStatesTechRep.pdf
+    It doesn't consider output feedback, specifically it models just the reservoir.
```

Comment:
This comment, without context to someone who doesn't know about ESNs, makes no sense.  Can you expand?